patientsearch/src/js/components/Utility.js

```diff
@@ -61,9 +61,18 @@ export function imageOK(img) {
   return true;
 }
 
-export function getUrlParameter(name) {
+export function getUrlParameter(name, queryString) {
   name = name.replace(/[\[]/, '\\[').replace(/[\]]/, '\\]');
   var regex = new RegExp('[\\?&]' + name + '=([^&#]*)');
-  var results = regex.exec(location.search);
+  var results = regex.exec(queryString ? queryString : location.search);
   return results === null ? '' : decodeURIComponent(results[1].replace(/\+/g, ' '));
 }
+
+export function getTimeZoneAdjustedDate(date) {
+  if (!date) return null;
+  return date.setMinutes(date.getMinutes() + date.getTimezoneOffset());
+}
```

Comment:
We may be able to add a default timezone to HAPI (PST instead of GMT). Let me know if that'd be helpful.

Reply:
Personally, I'd rather keep database level data in UTC.

Reply:
@ivan-c @pbugni just FYI, I didn't use this helper function in the code. I thought we might need it.  I did just remove the function as it is not used currently.